proteuslib/edb/tests/test_recarbonation_process_edb.py

```diff
@@ -0,0 +1,341 @@
+            for k in d1:
+                if k not in d2:
+                    result.append(f"{pfx}{k} in first, not in second")
+            for k in d2:
+                if k not in d1:
+                    result.append(f"{pfx}{k} in first, not in second")
+        for k in d1:
+            if k in d2:
+                pfx += f"{k}."
+                dict_diff(d1[k], d2[k], result=result, pfx=pfx)
+    return result
+
+
+@pytest.fixture
+def solver():
+    return get_solver()
+
+
+@pytest.fixture
+def equilibrium_config(thermo_config, reaction_config):
+    # Create a pyomo model object
+    model = ConcreteModel()
+    model.fs = FlowsheetBlock(default={"dynamic": False})
+
+    print(f"thermo_config components: {thermo_config['components'].keys()}")
+
+    model.fs.thermo_params = GenericParameterBlock(default=thermo_config)
+
+    model.fs.rxn_params = GenericReactionParameterBlock(
+        default={"property_package": model.fs.thermo_params, **reaction_config}
+    )
+
+    model.fs.unit = EquilibriumReactor(
+        default={
+            "property_package": model.fs.thermo_params,
+            "reaction_package": model.fs.rxn_params,
+            "has_rate_reactions": False,
+            "has_equilibrium_reactions": True,
+            "has_heat_transfer": False,
+            "has_heat_of_reaction": False,
+            "has_pressure_change": False,
+        }
+    )
+
+    model.fs.unit.inlet.flow_mol.fix(10)
+    model.fs.unit.inlet.pressure.fix(101325.0)
+    model.fs.unit.inlet.temperature.fix(300.0)
+    model.fs.unit.inlet.mole_frac_comp[0, "H +"].fix(0.0)
+    model.fs.unit.inlet.mole_frac_comp[0, "OH -"].fix(0.0)
+    model.fs.unit.inlet.mole_frac_comp[0, "H2CO3"].fix(0.0)
+    model.fs.unit.inlet.mole_frac_comp[0, "HCO3 -"].fix(0.0)
+    model.fs.unit.inlet.mole_frac_comp[0, "CO3 2-"].fix(0.0)
+    model.fs.unit.inlet.mole_frac_comp[0, "CO2"].fix(0.0005)
+    model.fs.unit.inlet.mole_frac_comp[0, "H2O"].fix(1 - 0.0005)
+
+    print(
+        f"thermo_params components: {model.fs.thermo_params.component_list.value_list}"
+    )
+    return model
+
+
+@pytest.mark.skip
+@pytest.mark.unit
+def test_build_model_equilibrium(equilibrium_config):
+    model = equilibrium_config
+
+    assert hasattr(model.fs.thermo_params, "component_list")
+    assert len(model.fs.thermo_params.component_list) == 7
+    assert "H2O" in model.fs.thermo_params.component_list
+    assert isinstance(model.fs.thermo_params.H2O, Solvent)
+    assert "H +" in model.fs.thermo_params.component_list
+    assert isinstance(model.fs.thermo_params.component("H +"), Cation)
+    assert "OH -" in model.fs.thermo_params.component_list
+    assert isinstance(model.fs.thermo_params.component("OH -"), Anion)
+
+    assert hasattr(model.fs.thermo_params, "phase_list")
+    assert len(model.fs.thermo_params.phase_list) == 2
+    assert isinstance(model.fs.thermo_params.Liq, AqueousPhase)
+    assert isinstance(model.fs.thermo_params.Vap, VaporPhase)
```

Comment:
Are these tests being skipped because the model is not building correctly? We may want to schedule time for the whole chem team to discuss if there are issues here.